src/plugins/notebook/components/NotebookEmbed.vue

```diff
@@ -79,24 +85,40 @@ export default {
                 : this.embed.snapshot.src;
         }
     },
+    watch: {
+        isLocked(value) {
+            if (value === true) {
```

Comment:
Will value never be equal to `false` here? Do we want to handle that case and enable removeEmbed?

Reply:
I don't believe it will. I think the idea is, once it's locked, it's locked. It would take an admin to unlock it and at that point, it would be mounted in an unlocked state, we only need to watch for it being locked, live.